src/compositions/useFlowRunsCount.ts

```diff
@@ -1,31 +1,39 @@
 import { SubscriptionOptions, UseSubscription, useSubscriptionWithDependencies } from '@prefecthq/vue-compositions'
-import { ComputedRef, Ref, computed, ref } from 'vue'
+import merge from 'lodash.merge'
```

Comment:
Here and elsewhere we should be using bucket imports for lodash
```suggestion
import { merge } from 'lodash'
```
https://lodash.com/per-method-packages

Reply:
I would agree if we had lodash itself installed and I understand that in v5 the individual packages will be deprecated and removed. But currently we have individual v4 packages installed and I actually believe the import you're suggesting here will break in production.